build.gradle

```diff
@@ -125,6 +125,8 @@ subprojects {
                 "net.sourceforge.htmlunit:htmlunit:2.28",
                 "com.github.stefanbirkner:system-rules:1.19.0",
                 "com.github.fppt:jedis-mock:0.1.9",
+                "it.ozimov:embedded-redis:0.7.2",
```

Comment:
Do we need this when we already have `"com.github.fppt:jedis-mock:0.1.9",`?

Reply:
So the original story was about making sure that redis itself could handle certain characters. Jedis, as far as I can tell, is just a mock for redis and uses a Table under the hood for storage. Embedded redis uses an actual redis executable so in theory would properly test if redis can handle said characters. However, as we're storing certs in base64 it might be a moot point.